src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_docker_utils.py

```diff
@@ -28,6 +28,9 @@
 
 
 ACCESS_TOKEN_PERMISSION = ['*', 'pull']
+IMAGE_BY_TAG_HELP = "The name of the image. May include a tag in the format 'name:tag'."
+IMAGE_BY_TAG_OR_DIGEST_HELP = "The name of the image. May include a tag in the format 'name:tag' " \
```

Comment:
nit: if used as CLIError, doesn't read right. Change to: "The name of the image may include .. "